docs/source/command_line.rst

```diff
@@ -482,6 +482,23 @@ of the above sections.
            # 'items' now has type List[List[str]]
            ...
 
+.. option:: --local-partial-types
+
+    By default, mypy won't check partial types spanning module top level or class top level.
+    This flag changes the behavior to only allow partial types at local level. For example:
```

Comment:
We don't mention partial types anywhere else in the documentation, so they are essentially a low-level implementation concept. It would be better if the explanation would explain what the option does without referring to the partial types concept.

Also mention that this is always implicitly enabled in the mypy daemon and will become enabled by default in a future mypy release.

Reply:
Good catch, what do you suggest as an alternative for partial types that is more user-friendly to non-devs?

Reply:
My current thoughts are to keep the name 'partial types' unchanged in the docs, in correspondence with the option name, and provide some description for partial types, maybe like
`Types such as List[?] where type arguments are unknown or None` so the readers can connect the name with some concrete examples